### DISCUSSION TITLE
Add option for high sample rate replays

### DISCUSSION THREAD

**Discussion by redactdead** [ppy/osu#30935] [Ideas]
I would like the option to save higher sample rate replays for scores done locally.

To my knowledge, current replays capture inputs 60 times per second. While this is usually (but not always) enough to recreate a replay that matches the original, I think that sometimes it just doesn't look smooth enough, especially on high level / fast cursor movement gameplay.

Here's a comparison between a [livestream recording](https://www.youtube.com/watch?v=C96GPcTBfvw&t=261s) (red) and a [replay recording](https://osu.ppy.sh/scores/1729399635) (white) of a cursor (trail) during a spinner.
![image](https://github.com/user-attachments/assets/9e282135-41f0-4b96-bbac-4ada08a3eca3)

I can understand the storage concerns of increasing the size of replays for all submitted scores, which is why I only suggest this be an option to increase the rate for replays that are saved locally after setting a play (press `F2`).

**Comment by pompydev** [comment]
Since replay file size is proportional to capture rate, I feel like this would be a good supporter-only feature. [Kuuube](https://osu.ppy.sh/users/11160804) has compiled a pretty extensive [list](https://docs.google.com/spreadsheets/d/125LNzGmidy1gagwYUt12tRhrNdrWFHhWon7kxWY7iWU/edit?gid=854129046#gid=854129046) of tablets used by osu! players and based on it I'd say increasing the limit to 240Hz would cover most cases. We could use native polling rate for tablets with polling rate lower than that.
